h/notification/reply.py

```diff
@@ -100,12 +100,11 @@ def get_notification(
 
```

Comment:
This whole file is a total nightmare. I wonder if it should be rolled into some kind of notification service which includes the subscription and the generation of these at the same time?

Maybe baby steps?